### ISSUE TITLE
Ability to make verticalIndicator fixed in a place, but also dragable

### ISSUE THREAD

**Issue by FickleLife** [aeyrium/bezier-chart#86]
Firstly, thank you for a great plugin! 

I have an idea for the vertical indicator line, would be great if it could be considered:

1. Ability to have it fixed in one position, with the graph moving underneath it. The bubble on the vertical indicator would update with the current values of the below graph
2. A secondary feature would be the vertical indicator is also allowed to be touch dragged - let's assume it is in the middle of the graph, you wouldn't be able to get to the start and end values. 

Basically, wherever the vertical indicator is it shows the value underneath in the bubble, but the VI can be dragged horizontally anywhere across the x axis too.
